test/data/car.cpp.h

```diff
@@ -0,0 +1,312 @@
+/**
+ * \file Vehicle.h
+ */
+
+#ifndef Vehicle_h
+#define Vehicle_h
+
+
+
+
+class Vehicle {
+  private:
+  protected:
+  public:
+
+  public:
+    Vehicle()
+    {
+      // @todo
+    }
+    // Public methods
+
+    /**
+     *  @return String
+     */
+    String getType() {
+      // @todo 
+      return String();
+    }
+
+
+  // Protected methods
```

Comment:
same here, unnecessarily keys..

Reply:
Same, `hasProtectedMethods` does not work, so as `protected:` as to be explicit every time I choose to make it explicit for the others too.